website/views/issue.py

```
@@ -677,17 +677,30 @@ def form_valid(self, form):
+
+                screenshot_path = os.path.join("uploads", f"{unique_hash}")
+                reopen = default_storage.open(screenshot_path, "rb")
+                django_file = File(reopen)
+                obj.screenshot.save(
+                    f"{unique_hash}",
+                    django_file,
+                    save=True,
+                )
+            except ValidationError:
+                messages.error(self.request, "Invalid screenshot hash provided")
+            except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)